src/transceivercontroller/DefaultTransceiverController.ts

```diff
@@ -16,8 +16,42 @@ export default class DefaultTransceiverController implements TransceiverControll
 
   constructor(protected logger: Logger, protected browserBehavior: BrowserBehavior) {}
 
-  setEncodingParameters(_params: Map<string, RTCRtpEncodingParameters>): void {
-    return;
+  async setEncodingParameters(encodingParamMap: Map<string, RTCRtpEncodingParameters>): Promise<void> {
+    if (!this._localCameraTransceiver || this._localCameraTransceiver.direction !== 'sendrecv') {
+      return;
+    }
+
+    const sender = this._localCameraTransceiver.sender;
+    const newEncodingParams = Array.from(encodingParamMap.values());
+    if (newEncodingParams.length <= 0) {
```

Comment:
Never less than zero, and you can check the size of `encodingParamMap` instead of turning it into an array first.